index.js

```diff
@@ -0,0 +1,64 @@
+const books = [
+  {
+    title: 'first',
+    author: 'name',
+  },
+  {
+    title: 'second',
+    author: 'name',
+  },
+  {
+    title: 'third',
+    author: 'name',
+  },
+];
+
+// form declaration
+
+const form = document.querySelector('#form');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const formButton = document.querySelector('#submit-button');
+const bookList = document.querySelector('ul');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  form.submit();
+});
+
+// load each book form the local array
+
+const removeBook = (bookItem) => {
+  const removeBtn = bookItem.querySelector('.remove');
+  removeBtn.addEventListener('click', () => {
+    bookItem.remove();
+  });
+};
+
+window.onload = () => {
+  books.forEach((book) => {
+    const bookItem = document.createElement('li');
+    bookItem.innerHTML = `${book.title} by ${book.author} <button class="remove">Remove</button>`;
+    bookList.appendChild(bookItem);
+    removeBook(bookItem);
+  });
+};
+
+const addBookToList = (books) => {
+  const bookItem = document.createElement('li');
+  bookItem.innerHTML = `${books.title} by ${books.author} <button class="remove">Remove</button>`;
+  bookList.appendChild(bookItem);
+  removeBook(bookItem);
+};
+
+formButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  const newBook = {
+    title: title.value,
+    author: author.value,
+  };
+
+  books.push(newBook);
+  localStorage.setItem('books', JSON.stringify(books));
+  addBookToList(newBook);
+});
```

Comment:
- [ ] Great effort working on your awesome book project, Currently your add and remove books from the list is functioning as expected but what i noticed here is the localStorage. I see that when a user add a book to the list, it is populated in the localStorage, but when it is deleted instead of the book should also be deleted from the localStorage, it still remain there.
- [ ] Also i notice when a book is added, it is stored in the localStorage but when i reload/refresh the page, the book vanished from the browser. but it still remain in the localStorage.

Below is the Video 👇 


https://user-images.githubusercontent.com/78634841/234151437-024a0ca1-a6a0-42be-8e10-4fae1083df56.mp4